### ISSUE TITLE
Produce a synthetic version of our governance without the notion of programs minimizing changes

### ISSUE THREAD

**Issue by mindthegab** [finos/community#45]
After #31, we need to produce a revamped version of our governance which does away with the concept of programs. We will take the occasion to ensure consistency across our policies at finos.org/governance, elements of governance organically grown in the [wiki](wiki.finos.org) and also to continue the process of moving out of the wiki into Github (per #36, we think this repository is the right place to host our governance moving forward, very much like CNCF is doing with their ["foundation" repository](https://github.com/cncf/foundation).

Foreseen activities will be:

- [x] Produce a short software project governance template that we can provide to projects, integrating it in the CONTRIBUTING.md of the https://github.com/finos/project-blueprint
- [x] Produce a short standard project governance template that we can provide to standards, integrating it in the CONTRIBUTING.md of the https://github.com/finos/project-blueprint or a parallel 'standard-blueprint'
- [x] Rework the active participation policy to have less no governance role in the projects, but still in the standards - so potentially this will be baked directly in the standard-blueprint
- [x] Add notion of Board approved Special Interest Groups (SIGs) with lightweight governance, e.g. maybe just a Chair / Vice-Chair (for redundancy)
- [x] Provide a "governance overview/entry point" in this repository, which outlines all "pan-project" governance items, including our Charter, linking to our policies, roles and responsibilities around the project lifecycle
- [x] Review if other governance related pages in the [Community Handbook](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/80642059/Community+Handbook) should be moved, looking at the sub-pages there:
     - [x] Any of the collaborative principles in the [Community Governance page](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530783/Community+Governance) 
     - [x] Anything from the [Decision Making page](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530351/Decision+Making)
     - [x] Anything from the [Project Level Processes page](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530309/Project-Level+Processes) 
     - [x] Anything from the [Working Group Level Processes page](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/80315025/Working+Group-Level+Processes), e.g. running meetings
     - [x] The [elections procedure](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/1124302852/Ranked-Choice+Voting+for+Board+Elections) - and maybe make it less brainiac in the process? :)
- [x] Review and approve https://github.com/finos/community/pull/51
Once we have agreement on these, and we have reviewed, we can then go back and do a proper cleanup of the wiki / community handbook, tracked in issue #48 

@copiesofcopies back to you - looking forward to reviewing this.

/cc @toshaellison @mcleo-d @brooklynrob - we'll ask for your review once we have an initial all-encompassing draft of this - which should be really soon.

**Comment by finos-admin** [comment]
@copiesofcopies for the "Standard Project" template, sounds like thfe LF is working on a very similar template for JDF-enabled standards (so with outlet to ISO, etc.) at 

https://github.com/CommunitySpecification/Working_Drafts/

Maybe worth connecting with folks there, and, for the make "standard project template" out of scope for this issue, and we can tackle it later once we know more about the LF work.

**Comment by copiesofcopies** [comment]
@finos-admin thanks for the heads up on the LF effort. I'll take a look and talk it over with Mike. In the meantime, here's a first draft of the other materials: https://github.com/copiesofcopies/community/tree/governance/governance

Note that there's no reference to the Active Participation Policy yet, as it's primarily relevant to standards projects under the proposed governance.

**Comment by mindthegab** [comment]
@copiesofcopies 

Sorry it took me so long to do a first pass, but here it is: please review [this PR](https://github.com/copiesofcopies/community/pull/1) which builds on your initial version. If / when you approve it I'd like to have @brooklynrob @maoo @toshaellison and @mcleo-d do a first step of review on your fork https://github.com/copiesofcopies/community/. 

To your question above re handbook, I've broken it down in the description of this issue, let me know if it works. 

I think we're very close, but before we can bring a PR back to the main repo (also for community review), I think we need to check all the boxes in the description. For the standard project, I'm happy if you have something initial that brings together what you had [here](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/1191903251/Running+a+Standards+Project) and then we can work on a v2 with the LF. 

Finally, I think we need an initial notion of SIGs, lightweight and Board approved. I've added a task in the description.

**Comment by copiesofcopies** [comment]
@mindthegab @brooklynrob Thoughts on how to capture (if at all) the "common collaborative principles" for projects listed here (and in the following section): https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530783/Community+Governance#CommunityGovernance-CommonCollaborativePrinciplesforProjectsandWorkingGroups. Right now, the best option I can think of is to make this a separate document (like the project lifecycle) in the governance folder and include one sentence linking to it in 1) the governance overview and 2) the project governance template. I don't know if it makes sense to include all of them in the template itself, particularly since projects can vary it.

**Comment by copiesofcopies** [comment]
@mindthegab I don't think it's worth including anything from the [Decision Making](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530351/Decision+Making) wiki page that's not already in the project governance template. I tried to simplify the voting process description as much as possible and I think it's sufficiently clear and explanatory.

**Comment by copiesofcopies** [comment]
@mindthegab I'm open to recommendations for the voting process. The problem I was trying to solve is that multiple-candidate Gold Director elections are prone to ties, because it's common to have almost as many candidates as there are voters. Ranked-Choice voting allows an "instant runoff" to be calculated based on the initial votes/rankings cast. There are two obvious alternatives: only elect one seat at a time (meaning multiple consecutive elections when there are multiple open seats) or have runoff votes in the event of ties that preclude a winner from being determined upon the initial vote.

**Comment by mindthegab** [comment]
> @mindthegab I'm open to recommendations for the voting process. The problem I was trying to solve is that multiple-candidate Gold Director elections are prone to ties, because it's common to have almost as many candidates as there are voters. Ranked-Choice voting allows an "instant runoff" to be calculated based on the initial votes/rankings cast. There are two obvious alternatives: only elect one seat at a time (meaning multiple consecutive elections when there are multiple open seats) or have runoff votes in the event of ties that preclude a winner from being determined upon the initial vote.

We don't have to change it right now - it was more as I thought both me and you were a bit confused when we actually used it.

OK to move it as is wherever you think it should live.

Having said that, I think the next big thing before we bring this to the Board (just for FYI) in two week, is closing on the standard template (even just a small rework of https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/1191903251/Running+a+Standards+Project for now) and the discussion on the SIGs.

When do you think I can review the next iteration (assuming you don't have further open questions @copiesofcopies? Also happy to jump on a call today to get started.

**Comment by copiesofcopies** [comment]
@mindthegab I will work on the standards project template today, but it's a bigger project than you have in mind. The document you linked to is just a plain-language summary of the IP Policy requirements re: standards projects. It does not define the governance of the standards projects themselves. To give you an idea of what a standards process document looks like, take a look at the [IEEE 802 working group's policy](http://www.ieee802.org/PNP/approved/IEEE_802_WG_PandP_v19.pdf). I'm confident ours can be significantly shorter, but this is already a relatively compact specimen.

**Comment by copiesofcopies** [comment]
@mindthegab I moved the software and standards project-related documents into subfolders. I created a README for the standards project subfolder based on the "Running a Standards Project" wiki page. And I created a draft standards project template charter. Looking forward to your thoughts.

**Comment by copiesofcopies** [comment]
I've updated the repo to address these items:

>     * [x]  Produce a short software project governance template that we can provide to projects, integrating it in the CONTRIBUTING.md of the https://github.com/finos/project-blueprint
> 
>     * [x]  Produce a short standard project governance template that we can provide to standards, integrating it in the CONTRIBUTING.md of the https://github.com/finos/project-blueprint or a parallel 'standard-blueprint' -- **I know we talked about holding off on this, but I think the document I adapted from the JDF is adequate for now, so I'm leaving it in.**
> 
>     * [x]  Rework the active participation policy to have less no governance role in the projects, but still in the standards - so potentially this will be baked directly in the standard-blueprint -- **The standards process template limits voting to participants who have attended 1/2 of the preceding 4 meetings, consistent with JDF's templates.**
> 
>     * [x]  Add notion of Board approved Special Interest Groups (SIGs) with lightweight governance, e.g. maybe just a Chair / Vice-Chair (for redundancy) -- **What you added is sufficient**
> 
>     * [x]  Provide a "governance overview/entry point" in this repository, which outlines all "pan-project" governance items, including our Charter, linking to our policies, roles and responsibilities around the project lifecycle
> 
>     * [x]  Review if other governance related pages in the [Community Handbook](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/80642059/Community+Handbook) should be moved, looking at the sub-pages there:
>       
>       * [x]  Any of the collaborative principles in the [Community Governance page](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530783/Community+Governance) -- **Adapted here: https://github.com/copiesofcopies/community/blob/governance/governance/Collaborative-Principles.md**
>       * [x]  Anything from the [Decision Making page](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530351/Decision+Making) -- **Replaced by voting rules built into project governance templates**
>       * [x]  Anything from the [Project Level Processes page](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/75530309/Project-Level+Processes) -- **Portions adapted as the standards project README https://github.com/copiesofcopies/community/tree/governance/governance/Standards-Projects**
>       * [x]  Anything from the [Working Group Level Processes page](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/80315025/Working+Group-Level+Processes), e.g. running meetings -- **Adapted here and linked in README and project governance template: https://github.com/copiesofcopies/community/blob/governance/governance/Meeting-Procedures.md**
>       * [x]  The [elections procedure](https://finosfoundation.atlassian.net/wiki/spaces/FINOS/pages/1124302852/Ranked-Choice+Voting+for+Board+Elections) - and maybe make it less brainiac in the process? :) -- **Moved in current braniac format here: https://github.com/copiesofcopies/community/blob/governance/governance/Board-Election-Procedure.md. With the new Gold Director election rules in the charter, we can begin effectively staggering elections to prevent multi-seat elections if we want (though it would mean more frequent elections).**

**Comment by mindthegab** [comment]
Looks great. I reviewed and sent you a smaller PR back here https://github.com/copiesofcopies/community/pull/2. 

Only open items per our convo today:
- [ ] close out work on standard - again, let's just keep it simple and shoot for a LF/JDF standard process in Oct 
- [ ] maybe remove documents that are unused (e.g. Charter.md, etc.)
- [ ] prep 1 slide in Board deck as overview of this work :)

Thanks!

**Comment by copiesofcopies** [comment]
@mindthegab I updated the [markdown version of the charter](https://github.com/copiesofcopies/community/blob/governance/governance/Charter.md) to improve the layout while retaining the original numbering. It's a bit of a hack (I used block quotes to indent the third level of sections) but I think it works. Let me know whether you prefer to leave this or the PDF in the folder.

I also updated the Standards Projects [README](https://github.com/copiesofcopies/community/tree/governance/governance/Standards-Projects) to include a recommendation of the Community Specification process. Take a look and see what you think.

I'll work on the slide now.

**Comment by copiesofcopies** [comment]
@mindthegab added the slide as well. Back to you.

**Comment by mindthegab** [comment]
Perfect - looks good to me. I will remove the PDF version for now and then I think we're ready to send a PR back to the community repo for review.

**Comment by mindthegab** [comment]
Hey @copiesofcopies - see my final review here https://github.com/copiesofcopies/community/pull/3.

Can you merge this and then merge the governance branch into master in your fork? 

That way we can than raise a PR against community:master and send that for team / community review.

**Comment by copiesofcopies** [comment]
Ok, merged your PR and merged governance into master on my fork: https://github.com/copiesofcopies/community
